test/pytest/test_bridge.py

```diff
@@ -689,7 +689,8 @@ async def recv_one():
 
 
 @pytest.mark.asyncio
-async def test_large_upload(event_loop: asyncio.AbstractEventLoop, transport: MockTransport, tmp_path: Path) -> None:
+async def test_large_upload(transport: MockTransport, tmp_path: Path) -> None:
+    event_loop = asyncio.get_running_loop()
```

Comment:
Pretty sure we don't have this on 3.6?

Reply:
Argh, good point, but tests pass? Don't we run tox on 3.6 here?

Reply:
Checked docs, introduced in 3.7

Reply:
I think we might be polyfilling this somewhere.  Seems a bit weird to depend on it, though...

Reply:
Agreed, but can we ignore deprecations with "required for 3.6"?

Reply:
Agreed, this is weird. Dropping this PR.